lib/global-styles.php

```diff
@@ -49,7 +49,6 @@ function gutenberg_experimental_global_styles_get_stylesheet( $tree, $type = 'al
  */
 function gutenberg_experimental_global_styles_enqueue_assets() {
 	if (
-		! get_theme_support( 'experimental-link-color' ) && // link color support needs the presets CSS variables regardless of the presence of theme.json file.
```

Comment:
I'm thinking of this use case:

- a theme doesn't have a `theme.json` but it opts-in to link color via `add_theme_support('experimental-link-color')`
- the UI control will be available to the user, as a result, the block can have a `style.color.link` attribute which will be rendered by the element support mechanism

At this point, if the user selected one of the presets, none will be available. It looks like we still need to do these checks.

---

For further thoughts (not this PR): what's a good path to deprecate the existing `add_theme_support` mechanism?

Reply:
Good point I reverted the change.

Reply:
ok, I've tested that this works in the front-end of TT1 as well. It doesn't work on the editor for for a different reason.